docs/command-reference/acl/save.md

```diff
@@ -0,0 +1,25 @@
+---
+description: Persist currently defined ACL's to file
+---
+
+# ACL SAVE
+
+## Syntax
+
+    ACL SAVE
+
+**ACL categories:** @admin, @slow, @dangerous
+
+When Dragonfly is configured to use an ACL file (with the `--aclfile` configuration option), this command will save the currently defined ACLs state to the ACL file.
+
```

Comment:
Pls add a note mentioning to make sure that dragonfly process has enough permissions to  write into`--aclfile` path.

Reply:
+1, I also added the `/etc/` read only in the managing dragonfly acl page. See: https://github.com/dragonflydb/documentation/pull/177/files#diff-be21c79345077536478605edbd6bc1bb24e287502ade8e4f15bb683c7d189dafR111